integration/airflow/tests/integration/tests/docker-compose.yml

```diff
@@ -39,9 +39,15 @@ x-airflow-base: &airflow-base
     SNOWFLAKE_ACCOUNT_ID: ${SNOWFLAKE_ACCOUNT_ID}
     SNOWFLAKE_USER: ${SNOWFLAKE_USER}
     SNOWFLAKE_PASSWORD: ${SNOWFLAKE_PASSWORD}
+    AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+    AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+    AWS_DEFAULT_REGION: ${AWS_DEFAULT_REGION}
+    AWS_ATHENA_OUTPUT_LOCATION: ${AWS_ATHENA_OUTPUT_LOCATION}
+    AWS_ATHENA_SUFFIX: ${AWS_ATHENA_SUFFIX}
```

Comment:
I think we don't have AWS account for tests right now. Maybe we could merge this PR with this test existing in a codebase, but not enabled? @sekikn

Reply:
@mobuchowski I already tested this PR using my AWS accound and it passed for Airflow 2.1.4, 2.2.4 and 2.3.4. So I think we can merge it and set AWS account on CircleCI later :+1:
https://app.circleci.com/pipelines/github/sekikn/OpenLineage/118/workflows/76821bea-11ec-4d90-91a2-421ca2554bec/jobs/2702?invite=true#step-109-1756
https://app.circleci.com/pipelines/github/sekikn/OpenLineage/118/workflows/76821bea-11ec-4d90-91a2-421ca2554bec/jobs/2706?invite=true#step-109-1739
https://app.circleci.com/pipelines/github/sekikn/OpenLineage/118/workflows/76821bea-11ec-4d90-91a2-421ca2554bec/jobs/2703?invite=true#step-109-1791